foca/api/register_openapi.py

```diff
@@ -39,7 +39,15 @@ def register_openapi(
     # Iterate over OpenAPI specs
     for spec in specs:
         spec_modified = False
-        spec_parsed = ConfigParser.parse_yaml(spec.path)
+
+        spec_parsed = dict()
+
+        if len(spec.path) > 1:
```

Comment:
Couldn't we simplify this like this:

```
if len(spec.path) > 1:
    spec_modified = True
spec_parsed = ConfigParser.merge_yaml(*spec.path)
```

Don't see why this wouldn't work with a list of size one?

Reply:
Because this code (from `.merge_yaml()`) does never enter the `for` loop if there's only one spec and thus return the parsed specs without any merging, which is exactly what we want:

```py
        args_list = list(args)
        if not args_list:
            return None
        yaml_dict = Addict(ConfigParser.parse_yaml(args_list.pop(0)))

        for arg in args_list:
            yaml_dict.update(Addict(ConfigParser.parse_yaml(arg)))

        return yaml_dict.to_dict()
```